keyboards/splitkb/kyria/keymaps/travishi/keymap.c

```diff
@@ -0,0 +1,575 @@
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+char wpm_str[10];
```

Comment:
Shouldn't be needed, nor used.
```suggestion
```

Reply:
This keeps cropping up, where are people getting it from?

Reply:
Noted, will remove and test. @keyboard-magpie from memory it might have been something that I thought I had to add for either the LED backlighting configuration or the Animation. Either way I'll remove and test.